test/server/overload_manager_impl_test.cc

```diff
@@ -339,6 +409,48 @@ TEST_F(OverloadManagerImplTest, DuplicateOverloadAction) {
                           "Duplicate overload action .*");
 }
 
+TEST_F(OverloadManagerImplTest, ScaledTriggerMaxLessThanMin) {
+  const std::string config = R"EOF(
+    resource_monitors {
+      name: "envoy.resource_monitors.fake_resource1"
+    }
+    actions {
+      name: "envoy.overload_actions.dummy_action"
+      triggers {
+        name: "envoy.resource_monitors.fake_resource1"
+        scaled {
+          scaling_threshold: 0.9
+          saturation_threshold: 0.8
+        }
+      }
+    }
+  )EOF";
+
+  EXPECT_THROW_WITH_REGEX(createOverloadManager(config), EnvoyException,
+                          "min_value must be less than max_value.*");
+}
+
+TEST_F(OverloadManagerImplTest, ScaledTriggerMaxEqualsMin) {
```

Comment:
Can you add one liner `//` above these tests explaining the intuition of what is being tested?